spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -87,30 +87,67 @@
 
   it "does not fail when the schema matches" do
     create_schema("array_schema", {
-      "type" => "array",
-      "items" => { "type" => "string" },
+      "type": "array",
+      "items": { "type": "string" },
     })
 
     expect(response_for(["valid"])).to match_response_schema("array_schema")
   end
 
   it "supports $ref" do
-    create_schema("single", {
-      "type" => "object",
-      "required" => ["foo"],
-      "properties" => {
-        "foo" => { "type" => "string" },
+    create_schema("user", {
+      "id": "file:/#{JsonMatchers.schema_root}/user.json#",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.